tests/data/ccm_example/data.1.json

```diff
@@ -0,0 +1,44 @@
+{
+    "metadata": {
+        "type": "LBDatafile",
+        "rank": 1
+    },
+    "phases": [
+        {
+            "id": 0,
+            "tasks": [
+                {
+                    "entity": {
+                        "home": 1,
+                        "id": 2,
+                        "migratable": true,
+                        "type": "object"
+                    },
+                    "node": 1,
+                    "resource": "cpu",
+                    "time": 5.0,
+                    "user_defined": {
+                        "shared_id": 1,
+                        "shared_bytes": 15000.0,
+                        "home_rank": 1
+                    }
+                }
+            ],
+            "communications": [
+                {
+                    "type": "SendRecv",
+                    "to": {
+                        "type": "object",
+                        "id": 1
+                    },
+                    "messages": 1,
+                    "from": {
+                        "type": "object",
+                        "id": 2
+                    },
+                    "bytes": 20000.0
+                }
+            ]
+        }
+    ]
+}
```

Comment:
missing EOF newline (same for other json files, not adding more comments)

Reply:
Fixed. (also restores unformatted json. it was formatted only for dev purpose)